lib/stm32wba/RF_Integration/STM32_WPAN/Target/linklayer_plat.c

```diff
@@ -63,26 +74,27 @@ uint32_t radio_sleep_timer_val;
   */
 void LINKLAYER_PLAT_ClockInit(void)
 {
-	AHB5_SwitchedOff = 0;
-	radio_sleep_timer_val = 0;
+#ifndef __ZEPHYR__
+  uint32_t linklayer_slp_clk_src = LL_RCC_RADIOSLEEPSOURCE_NONE;
 
-#ifdef __ZEPHYR__
-	LINKLAYER_PLAT_EnableBackupDomainAccess();
+  /* Get the Link Layer sleep timer clock source */
+  linklayer_slp_clk_src = LL_RCC_RADIO_GetSleepTimerClockSource();
+  if(linklayer_slp_clk_src == LL_RCC_RADIOSLEEPSOURCE_NONE)
+  {
+    /* If there is no clock source defined, should be selected before */
+    assert_param(0);
+  }
 #else
-	LL_PWR_EnableBkUpAccess();
-#endif
+	LINKLAYER_PLAT_EnableBackupDomainAccess();
 
 	/* Select LSE as Sleep CLK */
 	__HAL_RCC_RADIOSLPTIM_CONFIG(RCC_RADIOSTCLKSOURCE_LSE);
 
-#ifdef __ZEPHYR__
 	LINKLAYER_PLAT_DisableBackupDomainAccess();
-#else
-	LL_PWR_DisableBkUpAccess();
 #endif
 
-	/* Enable AHB5ENR peripheral clock (bus CLK) */
-	__HAL_RCC_RADIO_CLK_ENABLE();
+  /* Enable AHB5ENR peripheral clock (bus CLK) */
+  __HAL_RCC_RADIO_CLK_ENABLE();
```

Comment:
Keep tabs as indentation
```suggestion
	/* Enable AHB5ENR peripheral clock (bus CLK) */
	__HAL_RCC_RADIO_CLK_ENABLE();
```